vendor-android/src/main/kotlin/com/malinskiy/marathon/android/AndroidDevice.kt

```diff
@@ -18,7 +18,7 @@ class AndroidDevice(val ddmsDevice: IDevice) : Device {
     }
 
     override val model: String by lazy {
-        ddmsDevice.getProperty("ro.product.model")
+        ddmsDevice.getProperty("ro.product.model") ?: "Unknown"
```

Comment:
This is for the case that the device is not booted? Or even for a booted device?

Reply:
@Malinskiy i faced with this problem when i tried use custom emulator configurations